src/js/components/Meter/index.d.ts

```diff
@@ -11,13 +11,15 @@ export interface MeterProps {
   a11yTitle?: A11yTitleType;
   alignSelf?: AlignSelfType;
   background?: string | { color?: string; opacity?: OpacityType };
+  color?: string;
   gridArea?: GridAreaType;
   margin?: MarginType;
   max?: number;
   round?: boolean;
   size?: 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | 'full' | string;
   thickness?: 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | string;
   type?: 'bar' | 'circle';
+  value?: number;
```

Comment:
If I have a single value but want to have so `onHover` or `onClick` behavior, do I still have to use `values`? Or does this fall outside of the 80% use case where people just want to display a single value and no interaction is desired

Reply:
For that scenario, the default react/browser behavior of event listeners on the DOM element will work without grommet needing to do anything. Although, for a single value Meter, onClick might be better represented in the DOM via `<Button onClick={}><Meter /></Button>`.